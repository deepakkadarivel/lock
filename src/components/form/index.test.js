import React from 'react';
import { create } from 'react-test-renderer';
import Form from './index';

describe('Form component', () => {
    test('Renders Form component', () => {
        const component = create(
            <Form toggleDialog={() => {}} handleSubmit={() => {}}>
                <input type="text" name="name" />
            </Form>
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
