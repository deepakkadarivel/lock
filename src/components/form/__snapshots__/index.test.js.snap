// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component Renders Form component 1`] = `
<form
  className="Form"
  onSubmit={[Function]}
>
  <input
    name="name"
    type="text"
  />
  <div
    className="Form-footer"
  >
    <input
      className="Btn"
      onClick={[Function]}
      type="button"
      value="Cancel"
    />
    <input
      className="Btn Btn-primary"
      type="submit"
      value="Add"
    />
  </div>
</form>
`;
